src/Bootstrappers/BroadcastingConfigBootstrapper.php

```
@@ -64,17 +73,47 @@ public function bootstrap(Tenant $tenant): void
+
+        // Swap the currently bound Broadcaster singleton (resolved earlier with the central credentials)
+        // for the tenant BroadcastManager's default broadcaster, so that anything resolving the Broadcaster
+        // contract gets the same tenant broadcaster that the manager uses, instead of the stale central one.
+        $this->app->extend(Broadcaster::class, function () {
+            return $this->app->make(BroadcastManager::class)->connection();
+        });
+
+        // Extending the binding doesn't update the Broadcast facade's cached instance,
+        // so clear it to make the facade re-resolve to TenancyBroadcastManager instead of the central
+        // BroadcastManager — e.g. in the Broadcast::auth() call in BroadcastController (/broadcasting/auth).
+        Broadcast::clearResolvedInstance();
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\Broadcast' in method 'bootstrap'. (undefined)

(StaticAccess)